java/8/Dockerfile

```diff
@@ -1,7 +1,84 @@
+# docker build -t imqs/java-base:8-test --build-arg Maven_Nexus_User=$Maven_Nexus_User --build-arg Maven_Releases_Deployment=$Maven_Releases_Deployment --build-arg Maven_Snapshots_Deployment=$Maven_Snapshots_Deployment --build-arg Maven_Github_gerhardvanwyk=$Maven_Github_gerhardvanwyk .
+
 FROM imqs/ubuntu-base
 
-RUN apt-get update \
-	&& apt-get install openjdk-8-jre -y
+RUN apt update && \
+	apt install -y \
+		openjdk-8-jdk \

```

Comment:
Not sure why we need JDK? We won't be developing on top of this image?

Reply:
TL;DR: its adds 10mb to the image.

The idea is that the jar file gets created using this docker image. We do this with all our other services. It dramatically reduces the build job setups.